src/install/install.md

```diff
@@ -10,7 +10,7 @@ I can't recommend any particular installation method: they all have trade offs.
 
 ## Using pip
 
-System requirements:
+### System requirements:
```

Comment:
If this is a heading, then the actual installation steps should be another heading.  I think I'll change the current page a bit, and put the requirements in a grey box.